lib/SILOptimizer/LoopTransforms/ForEachLoopUnroll.cpp

```diff
@@ -21,7 +21,7 @@
 //    initializer annotated "array.uninitialized_intrinsic". This is done in
 //    `run` method.
 //
-// 2. For every "array.uninintialized_intrinsic" initializer call, try
+// 2. For every "array.unininitialized_intrinsic" initializer call, try
```

Comment:
Should be `uninitialized_intrinsic` instead?

Reply:
thanks for reviewing, fixed 🔨